acm/odh-edge/apps/bike-rental-app/bike-rental-inference-generator-cronjob.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: bike-rental-inference-generator
+spec:
+  schedule: "*/5 * * * *"  # This will run the job every hour
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: test-mlflow-container
+            image: quay.io/devguyio/ab-jq
```

Comment:
I invited you to the `rhoai-edge` org on Quay, maybe upload the image there?

![image](https://github.com/opendatahub-io/ai-edge/assets/99439005/4bac186a-6c76-41ae-a8b0-4a4222b85be9)

Reply:
Done